.github/workflows/event-push-to-integ.yml

```diff
@@ -7,40 +7,56 @@ on:
     branches:
       - master
       - '[0-9]+.[0-9]+'
-    paths: # Only run on changes to these paths (code, deps, and benchmarks related changes)
-      - 'src/**'
-      - 'deps/**'
-      - 'tests/benchmarks/**'
-      - '.github/workflows/benchmark-*.yml'
-      - '**/CMakeLists.txt'
-      - '**/Makefile'
 
 jobs:
-  ##### Remove path filter and uncomment the following lines if other jobs need to be triggered on push #####
+  benchmark-needed:
+    runs-on: ${{ vars.RUNS_ON || 'ubuntu-latest' }}
+    outputs:
+      BENCHMARK_NEEDED: ${{ steps.check-benchmarks.outputs.any_modified }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@
+        with:
+          fetch-depth: 0 # required for changed-files action to work
+      - name: Check if benchmarks were changed
+        id: check-benchmarks
+        uses: tj-actions/changed-files@
+        with: # Only run on changes to these paths (code, deps, and benchmarks related changes)
+          files: |
+            src/**
+            deps/**
+            tests/benchmarks/**
+            .github/workflows/benchmark-*.yml
+            **/CMakeLists.txt
+            **/Makefile
 
-  # benchmark-needed:
-  #   runs-on: ${{ vars.RUNS_ON || 'ubuntu-latest' }}
-  #   outputs:
-  #     BENCHMARK_NEEDED: ${{ steps.check-benchmarks.outputs.any_modified }}
-  #   steps:
-  #     - name: Checkout
-  #       uses: actions/checkout@
-  #       with:
-  #         fetch-depth: 0 # required for changed-files action to work
-  #     - name: Check if benchmarks were changed
-  #       id: check-benchmarks
-  #       uses: tj-actions/changed-files@
-  #       with: # Only run on changes to these paths (code, deps, and benchmarks related changes)
-  #         files: |
-  #           src/**
-  #           deps/**
-  #           tests/benchmarks/**
-  #           .github/workflows/benchmark-*.yml
-  #           **/CMakeLists.txt
-  #           **/Makefile
+  micro-benchmark-needed:
+    runs-on: ${{ vars.RUNS_ON || 'ubuntu-latest' }}
+    outputs:
+      MICRO_BENCHMARK_NEEDED: ${{ steps.check-benchmarks.outputs.any_modified }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@
+        with:
+          fetch-depth: 0 # required for changed-files action to work
+      - name: Check if micro-benchmarks were changed
+        id: check-benchmarks
+        uses: tj-actions/changed-files@
+        with: # Only run on changes to these paths (code, deps, and benchmarks related changes)
+          files: |
+            src/redisearch_rs/**
```

Comment:
Do we need to separate jobs? Can't we have a single job that checks out the code and then exports the required output variables in two different steps to trigger/suppress the benchmark/micro-benchmarks below?